src/authorizations/components/redesigned/ResourceAccordion.tsx

```diff
@@ -71,34 +73,47 @@ class ResourceAccordion extends Component<Props, State> {
             onToggleAll={this.handleToggleAll}
           />
           <GetResources resources={[ResourceType[resourceName]]}>
-            {resourceName === 'Telegrafs' ? (
-              <ResourceAccordionBody
-                resourceName={resource}
-                permissions={permissions[resource]}
-                onToggle={this.handleIndividualToggle}
-                title="Individual Telegraf Configuration Names"
-              />
-            ) : (
-              <ResourceAccordionBody
-                resourceName={resource}
-                permissions={permissions[resource]}
-                onToggle={this.handleIndividualToggle}
-                title="Individual Bucket Names"
-              />
-            )}
+            {isEmpty(permissions[resource].sublevelPermissions)
```

Comment:
alternatively, this might be a little clearer if it were written like this:

```ts
{!isEmpty(permissions[resource].sublevelPermissions) &&
  this.getAccordionBody(resourceName, resource)}
```

but your call